metadata.yaml

```diff
@@ -27,4 +27,4 @@ requires:
 storage:
   opensearch-data:
     type: filesystem
-    location: /var/snap/opensearch/common  # /mnt/opensearch/data
+    location: /var/snap/opensearch  # /mnt/opensearch/data
```

Comment:
this should be updated to comply with the spec but this can be addressed in a separated PR